.github/workflows/examples.yaml

```diff
@@ -134,8 +132,17 @@ jobs:
 
           ./builddir/example_meson_minimal_app
 
+      - name: Meson example (Windows)
+        if: matrix.config.label == 'windows'
+        run: |
+          python3 -m pip install meson ninja
+          cd examples/meson-minimal
+          meson setup builddir
+          meson compile -C builddir
+
+          start .\builddir\example_meson_minimal_app.exe
```

Comment:
There might be a smarter way to combine this step with the preceding one, but I couldn't get the github actions syntax quite right. 

Hopefully punt-able for now but let me know if any objection